.expeditor/config.yml

```diff
@@ -64,10 +64,8 @@ merge_actions:
 #
 # TODO: add promoted_to_unstable action to update changelog with modified omnibus components
 subscriptions:
-  - workload: artifact_published:unstable:supermarket:*
-    actions:
-      - built_in:update_acc
   - workload: artifact_published:stable:supermarket:*
     actions:
+      - built_in:update_acc
```

Comment:
Supermarket's pipeline in ACC is (was?) about going from unstable to stable, so I don't think moving this action has the desired result.

Reply:
Great catch @robbkidd. I fixed it so now `update_acc` is subscribed to promotion to `current`.